### DISCUSSION TITLE
Who has a good way to learn GitHub？

### DISCUSSION THREAD

**Discussion by yanghm123** [community/community#22593] [New to GitHub]
<p> GitHub newbie, the first time to report, many things do not understand, please take care of everyone in the future. Who has some good learning methods can also communicate and share.</p>

**Comment by MChevy422** [comment]
<p>Hello <a class="mention" href="/yanghm123">@yanghm123</a> ,</p>
<p>This post was moved to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our Community organized so users can more easily find information.</p>
<p>As you’ll notice, your Topic is now in the How to use Git and Github. No action is needed on your part; you can continue the conversation as normal here.</p>
<p>Let me know if you have any other questions or if I can help with anything else.</p>
<p>Greatly appreciated,</p>

**Comment by Wabri** [comment, marked as answer]
<p>There are a lot of way to learn git and in particularly github:</p>
<ul>
<li><a href="https://lab.github.com/" rel="nofollow noopener">https://lab.github.com/</a></li>
<li><a href="https://guides.github.com/" rel="nofollow noopener">https://guides.github.com/</a></li>
<li>if you want a video courses there is udemy</li>
<li>even on youtube and books</li>
</ul>
